src/main/java/org/apache/lucene/util/StringReaderCloner.java

```diff
@@ -18,9 +18,7 @@
  */
 
 // Using javax instead of java because of JVM security measures!
-package javax.io;
```

Comment:
No use to have this in javax.io, the class is not private. Now all ReaderCloners are in the same package.